packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -424,7 +423,6 @@ defineExpose({ input: computed(() => selectInput.value?.input) });
                     value: option.value,
                     label: option.label,
                     disabled: option.disabled,
-                    // TODO: remove type cast once its fixed in Vue / vue-tsc version
                     selected: arrayValue.some((value: TValue) => value === option.value),
```

Comment:
```suggestion
                    selected: arrayValue.some((value) => value === option.value),
```

I guess the TODO was meant for this one

Reply:
the type hint here is still necessary